Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -109,8 +109,12 @@ void Processor::initialize(u32)
         asm volatile("wfi");
 }
 
-void Processor::flush_tlb_local(VirtualAddress, size_t)
+void Processor::flush_tlb_local(VirtualAddress vaddr, size_t page_count)
```

Comment:
(it seems you use a different signature consistently in almost all the places; not sure if it begs for a inline proxy…? also `FlatPtr` should probably be use all along there—i guess that those are out of scope and weʼll just increase the crust a little! 😅)

Reply:
Yeah, really the callsites should be using VirtualAddresses instead of raw pointers instead, but that's once for the :yakstack: